src/features/help/components/dropDown.js

```diff
@@ -29,22 +29,22 @@ const StyledButton = styled.button`
   border-radius: 4px;
   line-height: 17px;
   height: 58px;
-  apperance: none;
+  appearance: none;
   padding: 20px;
   padding-left: 10px;
   text-indent: 10px;
   text-align: left;
   color: ${theme.colors.white};
   overflow: hidden;
-  appearance: none;
-  background: url(${props =>
-      props.isOpen ? `${chevronUp}` : `${chevronDown}`})
-    no-repeat 94%;
+  background-image: ${props =>
+    props.isOpen ? `url(${chevronUp})` : `url(${chevronDown})`};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline between consequent and alternate of ternary expression.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4765408)